sipi/scripts/sipi.init.lua

```diff
@@ -117,15 +124,17 @@ function pre_flight(prefix, identifier, cookie)
             -- server.log("pre_flight - restricted view settings - watermark: " .. tostring(response_json.restrictedViewSettings.watermark), server.loglevel.LOG_DEBUG)
```

Comment:
Is there a reason why this is commented out? I know it's not from you, but if the log has loglevel "Debug", I think we can as well activate it.

Reply:
true